xcp_d/utils/plot.py

```diff
@@ -1006,3 +1007,135 @@ def _get_tr(img):
     except AttributeError:  # Error out if not in cifti
         return img.header.get_zooms()[-1]
     raise RuntimeError("Could not extract TR - unknown data structure type")
+
+
+def plot_alff_reho_volumetric(output_path, filename, bold_file):
+    """Plot ALFF/ReHo mosaic plot for Niftis."""
+    import os
+
+    from bids.layout import parse_file_entities
+    from nilearn import plotting as plott
+    from templateflow.api import get as get_template
+
+    entities_to_use = ["cohort", "den", "res"]
+    space = parse_file_entities(bold_file)["space"]
+    file_entities = parse_file_entities(bold_file)
+    entities_to_use = {f: file_entities[f] for f in file_entities if f in entities_to_use}
+    template_file = get_template(template=space, **entities_to_use, suffix='T1w', desc=None)
+    if isinstance(template_file, list):
+        template_file = template_file[0]
+    template = str(template_file)
+    output_path = os.path.abspath(output_path)
+    plott.plot_stat_map(filename,
+                        bg_img=template,
+                        display_mode='z',
+                        cut_coords=8,
```

Comment:
```suggestion
                        display_mode='mosaic',
```

Since we bumped the pinned nilearn version, you can use mosaic here.

Reply:
sounds good!